src/components/Rating.jsx

```diff
@@ -72,6 +72,13 @@ export default class Rating extends Component {
   }
 
```

Comment:
I am also not very confident about 

```wrapperNode.value = value;```

It looks hacky. I will check other react component to find how they really handle this.